daemons/Makefile

```diff
@@ -17,4 +17,7 @@ dss-chunked-task:
 	./build_deploy_config.sh $@ $(DSS_DEPLOYMENT_STAGE)
 	cd $@; domovoi deploy --stage $(DSS_DEPLOYMENT_STAGE)
 
-.PHONY: dss-chunked-task dss-sync dss-index
+dss-gs-event-relay:
+	$(DSS_HOME)/scripts/deploy_gcf.py $@ --entry-point "dss_gs_bucket_events_$(subst -,_,$(DSS_GS_BUCKET))"
```

Comment:
IIRC, `test_sync.py` uses the test bucket. this will deploy the script against the dev bucket.

Reply:
`test_sync.py` is a unit test - it doesn't call live lambdas, calling the daemon entry points directly in-process instead. So it shouldn't care about which bucket the deployed event bus is connected to.

Reply:
Got it, thanks.